### PR TITLE
Disable distributed transactions test

### PR DESCRIPTION
**Pull request by roji**
See https://github.com/dotnet/runtime/issues/77241#issuecomment-1317114960

### CODE REVIEW

File: src/libraries/System.Transactions.Local/tests/OleTxTests.cs [hunk 1]
```diff
@@ -528,25 +528,26 @@ public void ImplicitDistributedTransactions_cannot_be_changed_after_being_set()
...
+//        using var _ = RemoteExecutor.Invoke(() =>
+//        {
+//            TransactionManager.ImplicitDistributedTransactions = true;
+//
+//            Test(MinimalOleTxScenario);
+//
+//            Assert.Throws<InvalidOperationException>(() => TransactionManager.ImplicitDistributedTransactions = false);
+//            TransactionManager.ImplicitDistributedTransactions = true;
+//        });
+//    }
```
Comment (stephentoub) [inline src/libraries/System.Transactions.Local/tests/OleTxTests.cs:550]: Can you just add `[ActiveIssue("https://github.com/dotnet/runtime/issues/77241")]` to it instead?

Comment (roji) [reply src/libraries/System.Transactions.Local/tests/OleTxTests.cs]: Thanks, I was looking for the way to do this in runtime...

Comment (roji) [reply src/libraries/System.Transactions.Local/tests/OleTxTests.cs]: (assuming it's OK to add this alongside [ConditionalFact]....)

Comment (stephentoub) [reply src/libraries/System.Transactions.Local/tests/OleTxTests.cs]: Yup

**Review by stephentoub — APPROVED** [review]

File: src/libraries/System.Transactions.Local/tests/OleTxTests.cs [hunk 2]
```diff
@@ -529,6 +529,7 @@ public void ImplicitDistributedTransactions_cannot_be_changed_after_being_set()
     }
 
     [ConditionalFact(nameof(IsRemoteExecutorSupportedAndNotNano))]
+    [ActiveIssue("https://github.com/dotnet/runtime/issues/77241")]
     public void ImplicitDistributedTransactions_cannot_be_changed_after_being_read_as_true()
```
Comment (jakobbotsch) [inline src/libraries/System.Transactions.Local/tests/OleTxTests.cs:533]: Should the `ImplicitDistributedTransactions_cannot_be_changed_after_being_read_as_false` right below this also be disabled?

Comment (roji) [reply src/libraries/System.Transactions.Local/tests/OleTxTests.cs:533]: I'm not seeing a test failure for that in #77241, did I miss another failure somewhere?

I agree that the failure likely isn't specifically related to ImplicitDistributedTransactions_cannot_be_changed_after_being_read_as_true, and part of the reason for disabling it is to see if the failure slips to another test, which may provide a bit more information... Any thoughts?

Comment (jakobbotsch) [reply src/libraries/System.Transactions.Local/tests/OleTxTests.cs:533]: I can't find any failures for that one either so that sounds good to me.

**Pull request merged**